ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -65,7 +65,7 @@ listen_addresses = '*'      # what IP address(es) to listen on;
                     # defaults to 'localhost', '*' = all
                     # (change requires restart)
 port = 5432             # (change requires restart)
-max_connections = 100           # (change requires restart)
+max_connections = 300           # (change requires restart)
```

Comment:
@snopoke do you think this number is actually too high? everytime postgres goes tits up we have to kill a bunch of connections. trying to do a bit of research into it, but it seems like a complicated question that depends on a variety of factors. http://serverfault.com/a/482302/246939
https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server